commit 2901c216a1d5867ac997e8c7805a3e530201fe71
Author: Kay Sievers <kay.sievers@vrfy.org>
Date:   Fri Jul 14 17:46:57 2006 +0000

    Replace dbus_connection_disconnect() with dbus_connection_close() (D-BUS
        api rename)

diff --git a/tools/hal-device.c b/tools/hal-device.c
index 06d6b22..a5b7c83 100644
--- a/tools/hal-device.c
+++ b/tools/hal-device.c
@@ -155,7 +155,7 @@ int main(int argc, char **argv)
 
 	libhal_ctx_shutdown(hal_ctx, &error);
 	libhal_ctx_free(hal_ctx);
-	dbus_connection_disconnect(conn);
+	dbus_connection_close(conn);
 	dbus_connection_unref(conn);
 	dbus_error_free(&error);
 
diff --git a/tools/lshal.c b/tools/lshal.c
index 8c05759..42e9907 100644
--- a/tools/lshal.c
+++ b/tools/lshal.c
@@ -705,7 +705,7 @@ main (int argc, char *argv[])
 	libhal_ctx_shutdown (hal_ctx, &error);
 	libhal_ctx_free (hal_ctx);
 
-	dbus_connection_disconnect (conn);
+	dbus_connection_close (conn);
 	dbus_connection_unref (conn);
 
 	if (show_device)
